src/mongocxx/test/spec/monitoring.cpp

```diff
@@ -136,9 +198,10 @@ apm_checker::event apm_checker::to_event(stdx::string_view s) {
         return apm_checker::event::kill_cursors;
     if (s.to_string() == "getMore")
         return apm_checker::event::get_more;
-    else
-        throw mongocxx::logic_error{error_code::k_invalid_parameter,
-                                    "unrecognized event {" + s.to_string() + "}"};
+    if (s.to_string() == "configureFailPoint")
+        return apm_checker::event::configure_fail_point;
+    throw mongocxx::logic_error{error_code::k_invalid_parameter,
```

Comment:
No need for an else if the previous if statements all return when entered. My IDE was warning me about this.